libs/blocks/share/inline-utils.css

```diff
@@ -0,0 +1,45 @@
+main > .section > .inline-wrapper {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 8px 0 12px;
+}
+
+main > .section > .inline-wrapper > .inline {
+  margin: 0;
+}
+
+main > .section.center > .inline-wrapper {
+  margin: 0 auto;
+  justify-content: center;
+  text-align: center;
+}
+
+main > .section.center > .inline {
+  margin: 0 auto;
+}
+
+main > .section > .inline-wrapper > .inline + .inline {
+  padding-left: 20px;
+  margin-left: 20px;
+  border-left: 1px solid #adadad;
+}
+
+@media screen and (max-width: 600px) {
+  main > .section > .inline-wrapper {
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  main > .section.center > .inline-wrapper {
+    align-items: center;
+  }
+
+  main > .section > .inline-wrapper > .inline + .inline {
+    border-left: none;
+    padding-left: 0;
+    margin-left: 0;
+    border-top: 1px solid #adadad;
+  }
+}
```

Comment:
I agree, This would be a great universal section style too 'inline' content and have the ability to pass justification as needed. Here's a future page example that could benefit from this. see the nemonic list. 

<img width="1030" alt="Screen Shot 2023-03-22 at 1 06 30 PM" src="https://user-images.githubusercontent.com/2086146/227011435-5d543d79-4965-4321-be56-c38811ff4b94.png">

Reply:
Sounds great~ Which folders should I move `inline-utils.css` and `inline-utils.js` to?

Reply:
@chrischrischris Does it make sense to rename this to `inline.js` and put it in `/libs/utils/`? That's sorta where I'm thinking.

Reply:
> I agree, This would be a great universal section style to 'inline' content and have the ability to pass justification as needed. Here's a future page example that could benefit from this. see the nemonic list.
> 
> <img alt="Screen Shot 2023-03-22 at 1 06 30 PM" width="1030" src="https://user-images.githubusercontent.com/2086146/227011435-5d543d79-4965-4321-be56-c38811ff4b94.png">

Agreed! I think this is a great use case for that new pattern that's popping up.